boilerplate/storybook/storybook.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { getStorybookUI, configure } from "@storybook/react-native"
+import SplashScreen from 'react-native-splash-screen'
```

Comment:
Linter is complaining about the single quotes, I guess in this boilerplate the standard is double quotes right ?

Reply:
Thanks! Will fix it :+1: